src/cloud/project/project-conf-files_services.md

```diff
@@ -166,7 +166,7 @@ The following table lists the services used in {{site.data.var.ece}} and their v
 {:.error-table}
 Service   |  Magento 2.4  |Magento 2.3  | Magento 2.2
 --------- | ------------- |-------------| ------------
-`elasticsearch` | 7.7.x | **Magento version 2.3.5 and later**— 5.2, 6.5, 6.8, 7.5, 7.6, 7.7<br>**Magento version 2.3.1 to 2.3.4**— 5.2, 6.5<br>**Magento version 2.3.0**— 5.2  | **Magento version 2.2.8 and later**— 5.2, 6.5 <br>**Magento version 2.2.0 to 2.2.7**— 5.2
+`elasticsearch` | 7.7 | **Magento version 2.3.5 and later**— 5.2, 6.5, 6.8, 7.5, 7.6, 7.7<br>**Magento version 2.3.1 to 2.3.4**— 5.2, 6.5<br>**Magento version 2.3.0**— 5.2  | **Magento version 2.2.8 and later**— 5.2, 6.5 <br>**Magento version 2.2.0 to 2.2.7**— 5.2
```

Comment:
Going to have to assume that all the 2.3 and 2.2 numbers are correct

BUT

These documents -
https://devdocs.magento.com/guides/v2.4/install-gde/prereq/elasticsearch.html
https://devdocs.magento.com/guides/v2.4/release-notes/backward-incompatible-changes/index.html

State that the version of Elasticsearch for 2.4.0 is 7.6.x - which contradicts the 7.7 on this document

Reply:
@meker12  Don't get me in trouble. I only meant to fix some numbering. 😆  Maybe this is a good time to fix the ES versions? I don't want to go that far if you already have something staged?

Reply:
I realise this is for Cloud but if the versions are different, should not mention be made of this to avoid possible confusion?

Eg the pages above say "You can use other versions at your discretion, but we recommend using the tested version of Elasticsearch." - so someone might question 7.7 if they have seen it?

Reply:
> @meker12 Don't get me in trouble. I only meant to fix some numbering. 😆 Maybe this is a good time to fix the ES versions? I don't want to go that far if you already have something staged?

@hguthrie - I think some versions of services on Cloud are higher than lowest supported, just that as the pages say "You can use other....." - someone might be confused as to why the Cloud versions are higher. Perhaps a generic statement saying "Cloud uses higher versions of services than mentioned in the Standard Requirements" ?

Reply:
I may be making a mountain out of a molehill but in my experience if there is a chance that someone can misinterpet documentation then they will. 

As Terry Pratchett said "Million-to-one chances crop up nine times out of ten"

Reply:
I don't disagree with the logic, but the Cloud environment will only support the versions listed here. The table represents the compatibility for Cloud environment and sample Cloud instance. And Cloud usually uses "latest".

I can certainly review the consistency of the Elasticsearch version information for Cloud support of 2.4.x.

Reply:
so specifically on Cloud, customers must use Elasticsearch 7.7 -- that's the version that is available on the Cloud infrastructure.  I just updated this based on guidance from Cloud engineering in PR --https://github.com/magento/devdocs/pull/7788

I agree that it would be helpful to add a statement that says the versions supported for Magento Commerce on Cloud platform might be different than Magento Commerce Enterprise or Open source.  How to say that gracefully?

Reply:
Okay - agreed